cmd/genji/dbutil/dump.go

```diff
@@ -128,32 +206,13 @@ func dumpTable(tx *genji.Tx, tableName string, w io.Writer) error {
 			u = " UNIQUE"
 		}
 
-		_, err = fmt.Fprintf(w, "CREATE%s INDEX %s ON %s (%s);\n", u, index.Info.IndexName, index.Info.TableName,
+		infos := stringutil.Sprintf("CREATE%s INDEX %s ON %s (%s);", u, index.Info.IndexName, index.Info.TableName,
```

Comment:
We can use `fmt.Fprintf` directly here, `stringutil.Sprintf` is only meant to be used in the core of Genji to reduce the size of compiled Wasm

Reply:
Good catch.